bower.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "reviews-service",
+  "dependencies": {
+    "ember": "lts-2-8",
+    "ember-qunit-notifications": "0.1.0",
+    "osf-style": "https://github.com/CenterforOpenScience/osf-style.git#1.3.0",
+    "font-awesome": "4.6.3",
+    "bootstrap-sass": "3.3.6",
```

Comment:
Can you remove all the dependancies below "bootstrap-sass"? We'll add them back in as need, as suggested by Barrett.